packages/libuild/libuild-plugin-swc/src/transform.ts

```diff
@@ -12,101 +18,117 @@ export interface SwcTransformOptions {
   useDefineForClassFields?: boolean;
   externalHelpers?: boolean;
   pluginImport?: ImportItem[];
+  lodash?: boolean;
```

Comment:
Can we use `tramsfornLodash` to make the option name consistent?

Reply:
I think it's ok.